bin/cpumask.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+'''Convert CPU list(s) to CPU masks'''
+
+import argparse
+from dataclasses import dataclass
+
+
+@dataclass
+class global_vars:
+    '''Global variables'''
+
+    args = None
+
+
+def process_options():
+    '''Define the CLI argument parsing options'''
+
+    parser = argparse.ArgumentParser(description="CPU mask creator")
+
+    parser.add_argument("--cpus",
+                        dest = "cpus",
+                        help = "Use one or more times to define CPU and/or CPU lists to include in the mask",
+                        action = "append",
+                        type = str)
+    
+    myglobal.args = parser.parse_args()
+
+
+def main():
+    process_options()
+
+    cpus = set()
+    
+    for cpu1 in myglobal.args.cpus:
+        for cpu2 in cpu1.split(","):
+            cpu3 = cpu2.split("-")
+            if len(cpu3) == 1:
+                cpus.add(int(cpu3[0]))
+            else:
+                for cpu4 in range(int(cpu3[0]), int(cpu3[1])+1):
+                    cpus.add(cpu4)
+
+    mask = 0
+    
+    for cpu in cpus:
+        bitmask = 1 << cpu
+        mask |= bitmask
+
+    print('bitmask={:b}'.format(mask))
+    print('hexmask={:x}'.format(mask))
```

Comment:
assuming you gonna grep the line you need....
```
 python3 cpumask.py --cpu 1,2 --cpu 4
bitmask=10110
hexmask=16
```
otherwise, could make hex or bit an option?

Reply:
honestly, its been so long since I wrote this and forgot to commit it I don't even remember anymore...but that does appear to be what I did:

https://github.com/perftool-incubator/bench-tracer/blob/main/tracer-client#L72

Reply:
:)